### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by sc369**

### CODE REVIEW

File: src/components/chat/ChatList.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                                        type="button"
+                                        className="btn btn-success"
+                                        onClick={() => {
+                                            this.props.history.push(`/messages/${message.id}/edit`);
+                                        }}
+                                    >Edit
+                                </button>
+                                    : null}
+
+                                <button
```
Comment (naienko) [inline src/components/chat/ChatList.js]: I shouldn't be able to delete any messages but my own.

File: src/components/chat/ChatList.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    Add Message
+               </button>
+
+                {this.props.messages.map(message =>
+                    <div key={message.id} className="card ">
+                        <div className="card-body">
+                            <h4 className="card-title">
+                                <div className="text"> {message.text} </div>
+                                <div className="userName"> {`${message.user.username}`}</div>
+                                <div className="timestamp"> {`${message.timestamp}`}</div>
```
Comment (naienko) [inline src/components/chat/ChatList.js]: for some reason not all the timestamps are rendering on the dash in the same format

**Review by naienko — DISMISSED** [review]

**Review by naienko — DISMISSED** [review]

**Comment by RDilly** [comment]
appearance of all other modules is breaking once chat is implemented.
needs to be fixed.

Chat functions are working great, tho

**Review by BrianBNeal — COMMENTED** [review]
Good but I wouldn't mind if the input for adding a new message was on the same screen with the actual chat history.

**Review by naienko — DISMISSED** [review]

**Review by naienko — DISMISSED** [review]

**Review by naienko — DISMISSED** [review]

**Review by naienko — DISMISSED** [review]

**Review by naienko — APPROVED** [review]

**Review by katfree — APPROVED** [review]

**Pull request merged**